cstr/src/lib.rs

```diff
@@ -92,7 +115,7 @@ pub unsafe fn parse_attribute_string(text: *const WideChar) -> String {
 ///
 /// to_widechar_ptr converts the input string to a null terminated string encoded in UTF-16.
```

Comment:
very minor nit, but we should update the comments to just generic WideChar istead of UTF16

Reply:
Not minor, I should have caught this.